app/assets/stylesheets/hyrax/_select_collection_type.scss

```diff
@@ -0,0 +1,45 @@
+.collectiontypes {
+  .modal-body {
+    padding: 0;
+  }
+
+  .modal-title {
+    font-size: $font-size-h3;
+  }
+}
+
+.select-collectiontype:first-child {
+  border-top: none;
+}
+
+.select-collectiontype {
+  border-top: 1px solid $table-border-color;
+  padding: 15px 15px 0;
+
+  label {
+    margin: 0;
+    width: 100%;
+  }
+
+  input[type="radio"] {
```

Comment:
@adamjarling How bad is this?  Can we ignore it?  Or is there an alternate suggestion you can make?

Reply:
I did it this way because it's not an area I'm familiar with. I mimicked it from work types, which were done the same way. If this needs to be changed, work types should change as well.

Reply:
In general it's always good to be specific with selectors, unless there is a reason for broad, generic styling.  If there's an `id` or `class` to use, this would be preferred.  I can't see this in the file anymore, but I'd make sure if this is in a `.scss` file, the `input[type="radio"]` is within another selector so it's not global.   Make sense?

Reply:
ie. make sure it is used like:
```
.someSpecificClass {
   input[type="radio"] {
      color: #000;
   }
}
```

Reply:
Isn't the `input[type="radio"]` selector is already nested within `.select-collectiontype`? If so, it seems appropriately scoped as is.

Although it's better to use a class as a selector, the alternative here is to add a class on the input just to avoid using the input as a selector, which seems a little overboard to me (though it's a judgement call).